pkg/intermediate/aggregate.go

```diff
@@ -331,6 +340,12 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 		}
 		if !correlationRequired {
 			aggregationRecord.ReadyToSend = true
+			// if no correlation is required for inter-Node record, K8s metadata will not be filled.
```

Comment:
Could you rephrase this? Also please explicitly mention that for Intra-node flows and ToExternal flows, we can set the MetaDataFilled as true by default.